src/main/java/com/inf2b/projectcodegeneration/controllers/TransactionController.java

```diff
@@ -0,0 +1,23 @@
+package com.inf2b.projectcodegeneration.controllers;
+
+import com.inf2b.projectcodegeneration.models.Transaction;
+import com.inf2b.projectcodegeneration.services.TransactionService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/transactions")
+public class TransactionController {
+
+    private final TransactionService transactionService;
+
+    public TransactionController(TransactionService transactionService){
+        this.transactionService = transactionService;
+    }
+
+    @PostMapping("/transactions")
+    public Transaction commitTransaction(@RequestMapping Transaction transaction){
```

Comment:
```suggestion
    public Transaction commitTransaction(@RequestBody Transaction transaction){
```
Je request een body als het goed is, Requestmapping word gebruikt voor de http path